src/index.php

```diff
@@ -1,3 +1,49 @@
 <?php
-echo "welcome at cronbeat";
+
+// Define application root directory
+define('APP_DIR', dirname(__DIR__));
+
+// Include only the necessary files
+require_once APP_DIR . '/src/classes/database.php';
+require_once APP_DIR . '/src/controllers/BaseController.php';
+require_once APP_DIR . '/src/controllers/SetupController.php';
+require_once APP_DIR . '/src/controllers/LoginController.php';
+require_once APP_DIR . '/src/controllers/DashboardController.php';
+
+use Cronbeat\Database;
+use Cronbeat\Controllers\SetupController;
+use Cronbeat\Controllers\LoginController;
+use Cronbeat\Controllers\DashboardController;
+
+// Parse the URL to get the controller name
+$uri = $_SERVER['REQUEST_URI'];
+$uri = trim($uri, '/');
+$uri = explode('/', $uri);
+$controllerName = !empty($uri[0]) ? $uri[0] : 'login';
```

Comment:
@jetbrains-junie get rid of the comment, use a function instead (e.g. "parseControllerFromUrl")